src/runtime_src/core/tools/xbmgmt2/flash/xmc.cpp

```diff
@@ -30,6 +30,7 @@
 #include "core/common/device.h"
```

Comment:
Copyright date (line 2) needs to be updated.

Reply:
comment addressed.